documentation/api/query/v4/pql.markdown

```diff
@@ -381,3 +381,32 @@ Combined with aggregate functions, you can effectively rollup of results and agg
 up values, for example to return how many facts exist for each fact name across all facts:
 
     facts[name, count(value)] { certname ~ 'web.*' group by name }
+
+## Paging
+PQL supports restriction of the result set via the standard SQL paging clauses
```

Comment:
Might want to refer to these as 'sql-like', since this is in no way an implementation of the sql standard.